components/category_menu_items/category_menu_items.tsx

```diff
@@ -83,11 +83,18 @@ const CategoryMenuItems = (props: Props): JSX.Element | null => {
     }
 
     const categoryMenuItems = filteredCategories.map((category: ChannelCategory) => {
+        let text = category.display_name;
+        if (category.type === CategoryTypes.FAVORITES) {
+            text = intl.formatMessage({id: 'sidebar.types.favorites', defaultMessage: 'Favorites'}).toLowerCase();
+        }
+        if (category.type === CategoryTypes.CHANNELS) {
+            text = intl.formatMessage({id: 'sidebar.types.channels', defaultMessage: 'Channels'}).toLowerCase();
+        }
         return {
```

Comment:
nit : extra line between return and above `}`
```suggestion
        }
        
        return {
```